packages/sdk/src/evm/functions/contract/types.ts

```diff
@@ -0,0 +1,21 @@
+import { NetworkInput, PrebuiltContractType } from "../../core/types";
+import { SDKOptions } from "../../schema/sdk-options";
+import { ThirdwebStorage } from "@thirdweb-dev/storage";
+import { ContractInterface } from "ethers";
+
+export type GetContractFromAbiParams = {
+  address: string;
+  abi: ContractInterface;
+  network: NetworkInput;
```

Comment:
I think I'd prefer for this to just become `chain` tbh?

Reply:
in terms of the actual input type I mean, param naming aside

Reply:
That would be ideal - for now I like this for complete compatibility with current SDK + I think being able to pass in an RPC URL is nice. Wdyt?